recipes/sdl2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "SDL2" %}
+{% set version = "2.0.4" %}
+{% set sha256_targz = "da55e540bf6331824153805d58b590a29c39d2d506c6d02fa409aedeab21174b" %}
+{% set sha256_zip = "08ba1de8bb4236d3aa1a94168b6e1a4f286bd21ca3437c11689d4e55f3a9cc89" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz  # [unix]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.tar.gz  # [unix]
+  sha256: {{ sha256_targz }}  # [unix]
+  fn:  {{ name }}-{{ version }}.zip  # [win]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.zip  # [win]
+  sha256: {{ sha256_zip }}  # [win]
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - autoconf  # [unix]
+
+test:
+  commands:
+    - sdl2-config --version  # [unix]
+
+about:
+  home: https://www.libsdl.org/index.php
+  license: Public Domain/ZLIB
+  license_file: COPYING.txt
```

Comment:
This path is relative to the source directory, so the `COPYING.txt` you are providing are not being used. Therefore, you can remove them.